utils/price-feed/src/feed/composable.rs

```diff
@@ -0,0 +1,91 @@
+use super::{Feed, FeedError, FeedResult};
+use crate::{
+	asset::Asset,
+	feed::{
+		composable_api, Exponent, FeedIdentifier, FeedNotification, Price, TimeStamp, TimeStamped,
+		TimeStampedPrice, CHANNEL_BUFFER_SIZE,
+	},
+};
+use futures::StreamExt;
+use sp_arithmetic::{FixedPointNumber, FixedU128};
+use std::collections::HashSet;
+use subxt::{ClientBuilder, DefaultConfig, PolkadotExtrinsicParams};
+use tokio::sync::mpsc;
+use tokio_stream::wrappers::ReceiverStream;
+
+pub struct ComposableFeed;
+
+impl ComposableFeed {
+	pub async fn start(
+		assets: &HashSet<(Asset, Asset)>,
+	) -> FeedResult<Feed<FeedIdentifier, Asset, TimeStampedPrice>> {
+		let (sink, source) = mpsc::channel(CHANNEL_BUFFER_SIZE);
+		// Notifiy feed started
+		sink.send(FeedNotification::Started { feed: FeedIdentifier::Composable })
+			.await
+			.map_err(|_| FeedError::ChannelIsBroken)?;
+		let api = ClientBuilder::new()
+			.build()
```

Comment:
Perhaps I need a URL parameter to help specify non local host and non default port.

Reply:
This should require the URL, otherwise how would the program know which RPC host to subscribe to?

Reply:
Added.